install.sh

```diff
@@ -376,6 +376,10 @@ EOS
 else
     install -m755 -d "$rdata"/saved_caches
     install -d -m755 "$rsystemd"/scylla-server.service.d
+
+    cat << EOS > "$rprefix"/scripts/scylla_sysconfdir.py
+SYSCONFDIR="$rsysconfdir"
```

Comment:
This is right, but by accident. The rule is that files under "$root" (or any "$rsomething") must only be referenced by the install script itself. Anything that is accessed by scylla runtime should not use "$root" since it may not exist.

I think the correct choice here is "$prefix/$sysconfdir".

In theory every runtime name would be "$prefix/foo" and every install-time name would be "$root/$prefix/foo", but in practice /etc files sometimes go under $prefix (nonroot install) and sometimes don't (root install), and this causes confusion.

I'll post a patch to document this.